packages/cli/cloud/src/config/local.ts

```diff
@@ -0,0 +1,59 @@
+import path from 'path';
+import os from 'os';
+import fs from 'fs';
+import XDGAppPaths from 'xdg-app-paths';
+
+const APP_FOLDER_NAME = 'com.strapi.cli';
+
+export const CONFIG_FILENAME = 'config.json';
+
+export type LocalConfig = {
+  token?: string;
+};
+
+function checkDirectoryExists(directoryPath: string) {
+  try {
+    return fs.lstatSync(directoryPath).isDirectory();
+  } catch (e) {
+    return false;
+  }
+}
+
+// Determine storage path based on the operating system
+export function getTmpStoragePath() {
+  const storagePath = path.join(os.tmpdir(), APP_FOLDER_NAME);
```

Comment:
We store a `.tar.gz` file in the tmp folder the time to upload it, this file should be deleted after the upload and we don't want the user to keep it in their storage system as it can take space for nothing. the TMP folder is regularly emptied, and the XDG paths aren't. 😅